themes/custom/ts_wrin/.nvmrc

```diff
@@ -0,0 +1 @@
+16
```

Comment:
Added a `.nvmrc` file to document the node version required for the front end build.